spec/features/editing_file_attachments/edit_file_attachment_file_spec.rb

```diff
@@ -3,7 +3,8 @@
     given_there_is_an_edition_with_featured_attachments
     when_i_go_to_edit_an_attachment
     and_i_add_a_unique_reference
```

Comment:
Minor: we could call this "attachment metadata" to save someone changing it later.

Reply:
Updated.